docs/production-deployment/cloud/audit-logging.mdx

```diff
@@ -95,59 +94,53 @@ The log sent to the Kinesis stream is JSON in the following format:
+:::note
 
-### Create an Audit Log sink
+The [`X-Forwarded-For`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For) format is a comma-separated list of IP addresses which should be evaluated from the last to the first, until meeting the first untrusted IP address of the list. This allows for instance to consider proxies in the path.
 
-1. In the Temporal Cloud UI, select **Settings**.
-1. On the **Settings** page, select **Integrations**.
-1. In the **Audit Logging** card, select **Configure Audit Logs**.
-1. On the **Audit Logging** page, choose your **Access method** (either **Auto** or **Manual**).
-   - **Auto:** Configure the AWS CloudFormation stack in your AWS account from the Cloud UI.
-   - **Manual:** Use a generated AWS CloudFormation template to set up Kinesis manually.
-1. In **Kinesis ARN**, paste the Kinesis ARN from your AWS account.
-1. In **Role name**, provide a name for a new IAM Role.
-1. In **Select an AWS region**, select the appropriate region for your Kinesis stream.
+Temporal provides the caller IP address in that format to allow customers to identify a caller IP address even if one (or more proxies) are in the network path to reach Temporal Cloud.
 
-If you chose the **Auto** access method, continue with the following steps:
+:::
 
-1. Select **Save and launch stack**.
-1. In **Stack name** in the AWS CloudFormation console, specify a name for the stack.
-1. In the lower-right corner of the page, select **Create stack**.
+## How to configure Audit Logging {#configure-audit-logging}
 
-If you chose the **Manual** access method, continue with the following steps:
+Audit logging can be configured in both AWS Kinesis
 
-1. Select **Save and download template**.
-1. Open the [AWS CloudFormation console](https://console.aws.amazon.com/cloudformation/).
-1. Select **Create Stack**.
-1. On the **Create stack** page, select **Template is ready** and **Update a template file**.
-1. Select **Choose file** and specify the template you generated in step 1.
-1. Select **Next** on this page and on the next two pages.
-1. On the **Review** page, select **Create stack**.
+* [AWS Kinesis Instructions](/cloud/audit-logging-aws)
 
 ## How to consume an Audit Log {#consume-an-audit-log}
 
-After you create an Audit Log sink, wait for the logs to flow into the Kinesis stream.
+After you create an Audit Log sink, wait for the logs to flow into the stream.
 You should see the first logs 2–10 minutes after you configure the sink.
 Subsequent logs arrive every 2 minutes if any actions occurred during that 2-minute window.
 
 :::note
 
-You must configure and implement your own consumer of the Kinesis stream.
-For an example, see [Example of consuming an Audit Log](#example-of-consuming-an-audit-log).
+You must configure and implement your own consumer of the stream.
 
 :::
 
+### Audit Log format
+
+The log sent to the Kinesis stream is JSON in the following format:
```

Comment:
This is the duplicate, which is not cloud-agnostic either since mention of Kinesis
We should remove one of the two, and just made the `caller_ip_address` comment change in the one we keep